modules/fundamental/src/advisory/endpoints/test.rs

```diff
@@ -59,7 +59,8 @@ async fn all_advisories(ctx: TrustifyContext) -> Result<(), anyhow::Error> {
     let (storage, _) = FileSystemBackend::for_test().await?;
 
     let app = actix_web::test::init_service(
-        App::new().configure(|config| configure(config, db, storage.clone())),
+        App::new()
+            .service(web::scope("/api").configure(|config| configure(config, db, storage.clone()))),
```

Comment:
This is only necessary due to the "/api" prefix in the test uri below, correct? Alternatively, we could remove the "/api" prefix there, I guess.

Reply:
True. However, that would mean we have two sets of URLs. One for testing, one for the actual server. Which I think we should avoid.

Reply:
Maybe. But within the context of a single test, adding a scope seems like unnecessary noise to me. I hope we can trust actix to resolve the path in the same way I trust DNS to resolve the hostname.